src/richie/apps/core/context_processors.py

```diff
@@ -68,6 +69,15 @@ def site_metas(request: HttpRequest):
         # Eg. https://my-cdn-user.cdn-provider.com/media/
         context["MEDIA_URL_PREFIX"] = ""
 
+    # Performance configurations
+    if urlparse(storage_url).hostname:
+        # Use dns-prefetch when using external media host, like a CDN.
+        context["MEDIA_URL_IS_EXTERNAL"] = True
+        # Optionally preconnect to the CDN
+        context["MEDIA_HOSTNAME_PRECONNECT"] = getattr(
+            settings, "MEDIA_HOSTNAME_PRECONNECT", False
+        )
```

Comment:
Can you please add this setting within `settings.py` of sandbox and cookiecutter template with value set to False by default ?
I think this will help newcomers to be aware about this settings.

Reply:
Done.
Added to `sandbox/settings.py` and to cookiecutter's `settings.py` within the `Production` class.